ci-operator/step-registry/rosa/sts/cluster/provision/rosa-sts-cluster-provision-commands.sh

```diff
@@ -126,8 +126,18 @@ if [[ "$DISABLE_WORKLOAD_MONITORING" == "true" ]]; then
   DISABLE_WORKLOAD_MONITORING_SWITCH="--disable-workload-monitoring"
 fi
 
-HYPERSHIFT_SWITCH=""
 SUBNET_ID_SWITCH=""
+if [[ "$ENABLE_BYOVPC" == "true" ]]; then
+  PUBLIC_SUBNET_IDs=$(cat ${SHARED_DIR}/public_subnet_ids | tr -d "[']")
+  PRIVATE_SUBNET_IDs=$(cat ${SHARED_DIR}/private_subnet_ids | tr -d "[']")
+  if [[ -z "${PUBLIC_SUBNET_IDs}" ]] || [[ -z "${PRIVATE_SUBNET_IDs}" ]]; then
+    echo -e "The public_subnet_ids and the the privated_subnet_ids are mandatory."
+    exit 1
+  fi
+  SUBNET_ID_SWITCH="--subnet-ids ${PUBLIC_SUBNET_IDs},${PRIVATE_SUBNET_IDs}"
+fi
+
+HYPERSHIFT_SWITCH=""
```

Comment:
We should put this at the end. otherwise, the others will overwrite the defined at here.